Northstar.Custom/mod/scripts/vscripts/sh_message_utils.gnut

```diff
@@ -16,6 +16,18 @@ global function NSCreateStatusMessageOnPlayer
 global function NSEditStatusMessageOnPlayer
 global function NSDeleteStatusMessageOnPlayer
 
+global enum ePlayerChoiceStatus
+{
+	UNKNOWN,
+	NOT_FOUND,				// no choice matching the key was found
+	NOT_FOUND_FOR_PLAYER,	// choice exists, but the player was not proposed with it
+
+	ONGOING,				// choice is currently proposed to the player, whom hasn't answered it yet
+	NOT_ANSWERED,			// player did not answer the choice (RUI timed out)
+	CHOICE_1,				// player selected first option
+	CHOICE_2				// player selected second option
+}
```

Comment:
Not sure what our stance about tabs for alignment is. All I can see is that it looks misaligned on GitHub cause it defaults to 1 tab == 8 spaces ^^"

![image](https://github.com/user-attachments/assets/29c4bf02-7aa5-4216-a216-0c1c786fecb0)

Reply:
Should I go with spacing here then?

Reply:
yup, as tabs are for indentation and spaces are for alignment

(I wish we just used spaces everywhere so this wouldn't be an issue...)